bin/run.sh

```diff
@@ -0,0 +1,164 @@
+rm -f cookies.txt
+
+# get disk size
+qemu-img info 'json: { "file.cookie": "vmware_soap_session='$cookie'", "file.sslverify": "off", "file.driver": "https", "file.url": "'$diskUri'", "file.timeout": 2000 }' > output.txt
+size=$(grep -Eow "[0-9]+\.[0-9]+[MG]" output.txt)'i'
+rm -f output.txt
+
+# we support only one disk
+PVCNAME=$VM_NAME-disk-1
+
+# Create job template
+tee template.yaml <<EOY
+apiVersion: v1
+kind: Template
+metadata:
+  name: v2v-job-template
+  annotations:
+    openshift.io/display-name: "KubeVirt v2v VM Import"
+    description: |
+      A template to trigger a v2v job in order to import a VM from a remote
+      source (like a libvirt instance of a file-system image) into KubeVirt.
+      Example
+      libvirt vpx://vcenter.example.com/Datacenter/esxi
+      ova http://example.com/my.ova
+
+parameters:
+- name: SOURCE_TYPE
+  description: "The VM source for this job (libvirt) see man virt-v2v"
+  value: "ova"
+- name: SOURCE_NAME
+  decsription: "The name of the VM to import (name or URL)"
+  value: "http://192.168.42.1/my.ova"
+- name: SOURCE_URI
+  description: "(Optional) The URI to connect to the remote instance"
+- name: SOURCE_USER
+  description: "(Optional) The user to the remote instance"
+- name: SOURCE_PASSWORD
+  description: "(Optional) The password to the remote instance"
+
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: kubevirt-privileged
+    namespace: default
+- kind: Job
+  apiVersion: batch/v1 
+  metadata:
+    name: v2v
+    namespace: default
+  spec:
+    backoffLimit: 1
+    template:
+      spec:
+        serviceAccountName: kubevirt-privileged
+        restartPolicy: Never
+        containers:
+        - name: v2v
+          image: quay.io/pkliczewski/v2v-job
+          args: ["/v2v-dst",
+                 "\${SOURCE_TYPE}"
+                 "\${SOURCE_NAME}",
+                 "\${SOURCE_URI}",
+                 "\${SOURCE_USER}",
+                 "\${SOURCE_PASSWORD}"]
+          env:
+          - name: "DEBUG"
+            value: "1"
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - name: kvm
+            mountPath: /dev/kvm
+          - name: volume-1
+            mountPath: /dst
+          - name: volume-2
+            mountPath: /var/tmp
+        volumes:
+        - name: kvm
+          hostPath:
+            path: /dev/kvm
+        - name: volume-1
+          persistentVolumeClaim:
+            name: $PVCNAME
+        - name: volume-2
+          persistentVolumeClaim:
+            name: temp
+- kind: PersistentVolumeClaim
+  apiVersion: v1
+  metadata:
+    name: $PVCNAME
+  spec:
+    accessModes:
+      - ReadWriteOnce
+    volumeMode: Filesystem
+    resources:
+      requests:
+        storage: $size
+- kind: PersistentVolumeClaim
+  apiVersion: v1
+  metadata:
+    name: temp
+  spec:
+    accessModes:
+      - ReadWriteOnce
+    volumeMode: Filesystem
+    resources:
+      requests:
+        storage: $size
```

Comment:
This need to be checked empirically whether we need exactly the same size as target pvc.